javascript/ql/src/semmle/javascript/dataflow/DataFlow.qll

```diff
@@ -1089,6 +1149,27 @@ module DataFlow {
     nd = TExceptionalFunctionReturnNode(function)
   }
 
+  /**
+   * INTERNAL. DO NOT USE.
+   *
+   * Gets the `PropRead` node corresponding to the value stored in the given
+   * binding pattern due to destructuring.
+   *
+   * For example, in `let { p: value } = f()`, the `value` pattern maps to a `PropRead`
+   * extracting the `p` property.
+   */
+  DataFlow::PropRead patternPropRead(BindingPattern value) {
```

Comment:
Can this be made `private`? It only seems to be used above in the same module.

Reply:
Indeed, now that `getRhsNode` no longer uses it